pkgs/games/flightgear/default.nix

```diff
@@ -73,7 +73,8 @@ stdenv.mkDerivation rec {
     description = "Flight simulator";
     maintainers = with maintainers; [ raskin ];
     platforms = platforms.linux;
-    hydraPlatforms = []; # disabled from hydra because it's so big
+    hydraPlatforms = [ ]; # disabled from hydra because it's so big
```

Comment:
Why are you adding a space above? That's not related to the upgrade. Please remove.

Reply:
yeah sorry, it's hard to fight auto-formatting, for most new files its ok, since many people use something, but for older files small things like this slip.